discovery-infra/download_logs.py

```diff
@@ -105,7 +105,7 @@ def download_logs(client: InventoryClient, cluster: dict, dest: str, must_gather
 
             if must_gather:
                 recreate_folder(os.path.join(output_folder, "must-gather"))
-                config_etc_hosts(cluster['name'], cluster['base_dns_domain'], cluster['api_vip'])
+                config_etc_hosts(cluster['name'], cluster['base_dns_domain'], get_api_vip_from_cluster(cluster))
```

Comment:
Isn't `cluster['api']` must be defined correctly when we ask to download the logs? Otherwise, the cluster itself is misconfigured.

Reply:
no, in None platform or single node. U don't have ip for vip

Reply:
You don't an API virtual IP or you don't have an API endpoint at all?
DHCP mode isn't running on none-platform and single-node modes?